DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/ContractDetails/ContractDetailsViewModel.cs

```diff
@@ -0,0 +1,116 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.ContractDetails
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Models;
+    using DigitalLearningSolutions.Data.Models.User;
+    using DigitalLearningSolutions.Web.Helpers;
+
+    public class ContractDetailsViewModel
+    {
+        public ContractDetailsViewModel(List<AdminUser> adminUsers, Centre centreDetails, int numberOfCourses)
+        {
+            Administrators = adminUsers.Count(a => a.IsCentreAdmin).ToString();
+            Supervisors = adminUsers.Count(a => a.IsSupervisor).ToString();
+
+            var trainers = adminUsers.Count(a => a.IsTrainer);
+            var cmsAdministrators = adminUsers.Count(a => a.ImportOnly);
+            var cmsManagers = adminUsers.Count(a => a.IsContentManager) - cmsAdministrators;
+            var contentCreators = adminUsers.Count(a => a.IsContentCreator);
+
+            Trainers = DisplayStringHelper.GenerateNumberWithLimitDisplayString(trainers, centreDetails.TrainerSpots);

```

Comment:
This is quite a mouthful.. I wonder if there is an easier to read name.. Maybe "FormatNumberWithLimit"?